### PR TITLE
add a dockerfile for the import bin

### PR DESCRIPTION
**Pull request by antoine-de**
this way we'll be able to push some mimir images in hub.docker.com and run mimir's job in container.

We might want to rename `Dockerfile` to `Docker_bragi` but it will likely break some Kisio's integration, so I think we should discuss it.

If that's ok for you I think we should set up some automated build in dockerhub. It might be better if it's in the kisio's organisation but if you don't want to do it, I can set it up in qwantreasearch's organisation.

the remaining question is how we're gonna name it.
* `bragi` ? `mimir_bragi` ?
* `mimir_import` ?

### CODE REVIEW

**Comment by prhod** [comment]
As discussed:
- @antoine-de should repair this Dockerfile
- then, @kinnou02  will find the right button to automate the build
Decided  name : `mimirsbrunn` et `bragi`

**Comment by kinnou02** [comment]
You need to use a multistage build if you want to be independent of the host building infrastructure

**Comment by antoine-de** [comment]
yup, it will be done as soon as I manage to get around openssl :rage:

**Comment by antoine-de** [comment]
the dockerfile is now self contained and can be used in hub.docker.com

there is now 2 dockerfile (warning, this will break the current docker integrations):

`docker build -t mimirsbrunn -f docker/Dockerfile_import . ` to build the import image

`docker build -t bragi -f docker/Dockerfile_bragi .`

Note: @kinnou02  you can also automatically build tagged image in hub.docker.com (but since we don't do releases, it might not be worth it to set it up).

**Comment by antoine-de** [comment]
Note: the docker images uses debian 8 (jessie) because I have not been able to use debian 9 due to openssl strange errors :flushed:

**Comment by antoine-de** [comment]
NOTE: the build fail due to another rustfmt update (the fmt update are a bit painfull, maybe we should stick with the stable branch ?)

should I run rust fmt ? (I don't want to have conflict on the other branches since there are quite the number of opened PR out there)

**Comment by pbougue** [comment]
Nope, no fmt! We'll have a look tomorrow into that PR.

File: .dockerignore [hunk 1]
```diff
@@ -1,3 +1,8 @@
 *
-!target/release/bragi
-!build_packages/*
+!src/
+!libs/
+!tests/
+!json/
+!Cargo*
+!build.rs
+!.git/
```
[the remark this answers is not in the document]
Comment (antoine-de) [reply .dockerignore]: I wanted to have an image as clean as possible, but if you prefer I can change it

File: docker/Dockerfile_import [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update \
+    && apt-get install -y libcurl3 libgeos-c1 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY --from=builder /srv/mimirsbrunn/target/release/osm2mimir /usr/bin/osm2mimir
+COPY --from=builder /srv/mimirsbrunn/target/release/cosmogony2mimir /usr/bin/cosmogony2mimir
+COPY --from=builder /srv/mimirsbrunn/target/release/bano2mimir /usr/bin/bano2mimir
+COPY --from=builder /srv/mimirsbrunn/target/release/openaddresses2mimir /usr/bin/openaddresses2mimir
```
[the remark this answers is not in the document]
Comment (antoine-de) [reply docker/Dockerfile_import]: oups, I'll add them

**Pull request merged**